easybuild/easyblocks/c/cblas.py

```diff
@@ -55,12 +56,18 @@ def configure_step(self):
         self.cfg.update('buildopts', 'FC="%s"' % os.getenv('F77'))
         self.cfg.update('buildopts', 'CFLAGS="%s -DADD_"' % os.getenv('CFLAGS'))
         self.cfg.update('buildopts', 'FFLAGS="%s -DADD_"' % os.getenv('FFLAGS'))
+
         blas_lib_dir = os.getenv('BLAS_LIB_DIR')
         blas_libs = []
         for blas_lib in os.getenv('BLAS_STATIC_LIBS').split(','):
             blas_lib = os.path.join(blas_lib_dir, blas_lib)
             if os.path.exists(blas_lib):
                 blas_libs.append(blas_lib)
+
+        if get_software_root("imkl"):
+            extra_blas_libs = '-lmkl_intel_thread -lmkl_lapack95_lp64 -lmkl_intel_lp64 -lmkl_core'
+            blas_libs += extra_blas_libs.split()
```

Comment:
This will include the `-l` which I think is wrong. Are you sure? Why not simply `extra_blas_libs = ['mkl_intel_thread', 'lmkl_lapack95_lp64' , ...]`

Reply:
thanks, I guess I got carried away :)